pages/nfttest/fetch/index.vue

```diff
@@ -188,15 +219,82 @@ export default class FetchIndex extends Vue {
     return string
   }
 
-  async submitToISCN(data: ISCNRegisterPayload): Promise<void> {
-    this.isLoading = true
+  async crawlUrlData() {
+    try {
+      const { data } = await this.$axios.get(`/crawler/?url=${encodeURIComponent(this.url)}`)
+      this.crawledData = data
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error('CANNOT_CRAWL_URL')
+      throw err
+    }
+  }
+
+  async estimateArweaveFee() {
+    try {
+      const { address, arweaveId, LIKE, ipfsHash, memo } = await this.$axios.$post(
+        API_POST_ARWEAVE_ESTIMATE,
+        this.formData,
+        {
+          headers: {
+            'Content-Type': 'multipart/form-data',
+          },
+        },
+      )
+      this.arweaveId = arweaveId
+      this.ipfsHash = ipfsHash
+      return {
+        to: address,
+        amount: new BigNumber(LIKE),
+        memo,
+      }
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error('CANNOT_ESTIMATE_ARWEAVE_FEE')
+      throw err
+    }
+  }
+
+  async sendArweaveFeeTx({to, amount, memo }: { to: string, amount: BigNumber, memo: string }): Promise<void> {
+    if (!this.signer) throw new Error('SIGNER_NOT_INITED')
+    if (!to) throw new Error('TARGET_ADDRESS_NOT_SET')
+    try {
+      const { transactionHash } = await sendLIKE(this.address, to, amount.toFixed(), this.signer, memo)
+      this.txHash = transactionHash
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error('CANNOT_SEND_ARWEAVE_FEE_TX')
+      throw err
+    }
+  }
+
+  async submitToArweave(): Promise<void> {
+    try {
+      const { arweaveId } = await this.$axios.$post(
+        `${API_POST_ARWEAVE_UPLOAD}?txHash=${this.txHash}`,
```

Comment:
I think `txHash ` can be passed as function param instead of this.data here, you don't use the hash any where else.
Also the name `txHash` is confusing